manifests/install/client/debian.pp

```diff
@@ -4,32 +4,20 @@
 # This code is needed as the --mkhomedir parameter passed to ipa-client-install does
 # not configure PAM even though it does install the required packages.
 #
-# Currently Ubuntu 14.04/16.04 and Debian 8/9 are supported.
-#
 class easy_ipa::install::client::debian {
-  case $facts['os']['distro']['codename'] {
-    /^(xenial|stretch|bionic|focal|buster|bullseye|jammy|bookworm)$/: {
-      # Ensure that required packages are present even if they do not get pulled
-      # in as freeipa-client package dependencies
-      stdlib::ensure_packages(['oddjob','oddjob-mkhomedir'], { 'ensure' => 'present' })
+  # Ensure that required packages are present even if they do not get pulled
+  # in as freeipa-client package dependencies
+  stdlib::ensure_packages(['oddjob','oddjob-mkhomedir'], { 'ensure' => 'present' })
```

Comment:
Given the length of the "supported operating system list" I think this change is reasonable.